bitnami/mariadb/templates/secrets.yaml

```diff
@@ -1,9 +1,11 @@
 {{- $host := include "mariadb.primary.fullname" . }}
 {{- $port := print .Values.primary.service.ports.mysql }}
 {{- $rootPassword := include "common.secrets.passwords.manage" (dict "secret" (include "mariadb.secretName" .) "key" "mariadb-root-password" "providedValues" (list "auth.rootPassword") "context" $) | trimAll "\"" | b64dec }}
-{{- $password := .Values.auth.password }}
-{{- if eq (include "mariadb.secret.existPassword" .) "true" }}
-{{- $password := include "common.secrets.passwords.manage" (dict "secret" (include "mariadb.secretName" .) "key" "mariadb-password" "providedValues" (list "auth.password") "context" $) | trimAll "\"" | b64dec }}
+{{- $password := .Values.auth.password  }}
```

Comment:
In this case why don't you encode the password with b4enc?

Reply:
Here we are managing the password in plain. The encoding is applied some lines below. I know it could be improved and there is an internal task to achieve it.